src/command/restore/restore.c

```diff
@@ -1232,11 +1279,14 @@ restoreSelectiveExpression(Manifest *manifest)
                     includeDb = varStrForce(VARUINT(db->id));
                 }
 
+                // Error if the db id is in the exclude list
+                if(strLstExists(excludeDbIdList, includeDb))
+                    THROW_FMT(DbInvalidError, "database to include '%s' is in the exclude list", strZ(includeDb));
+
                 // Error if the db is a system db
-                if (cvtZToUInt64(strZ(includeDb)) < PG_USER_OBJECT_MIN_ID)
+                if (cvtZToUInt64(strZ(includeDb)) < PG_USER_OBJECT_MIN_ID || strLstExists(systemDbIdList, includeDb))
                     THROW(DbInvalidError, "system databases (template0, postgres, etc.) are included by default");
 
-                // Remove from list of DBs to zero
```

Comment:
Can we add back this comment? The comment at the beginning of the loop is great, but also keeping this causes less churn in the diff's when reviewing - I had to ask myself "Wait, why was this removed?"

Reply:
Ok